routes/users.js

```diff
@@ -8,5 +10,9 @@ userRoute.put('/change-password/:id', (changeUserPassword));
 userRoute.patch('/:id/:token', (resetUserPassword));
 userRoute.post('/forgot-password', (userForgotPassword));
 userRoute.post('/login', (userLogin));
+//userRoute.get('/', authenticate, (getAllUsers));
+//userRoute.delete('/:userId', authenticate, (terminateUser));
+userRoute.get('/:recipeId', userAuthenticate, (getARecipe));
+userRoute.get('/', userAuthenticate, (getAllRecipess));
```

Comment:
```suggestion
userRoute.get('/', userAuthenticate, (getAllRecipes));
```

Reply:
I was thinking about separating both authentication and calling them in both routes instead of in a particular route